md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/DeployStatusAPI.java

```diff
@@ -66,11 +64,11 @@ RestWrapper get(
                 deployStatus.setDeployStatusId((int) jpaDeployStatus.getDeployStatusId());
                 deployStatus.setDescription(jpaDeployStatus.getDescription());
             }
-            // deployStatus = s.selectOne("call_procedures.GetDeployStatus", deployStatus);
             restWrapper = new RestWrapper(deployStatus, RestWrapper.OK);
-            LOGGER.info("Record with ID:" + deployStatusId + " selected from DeployStatus by User:" + principal.getName());
-        } catch (Exception e) {
-            restWrapper = new RestWrapper(e.getMessage(), RestWrapper.ERROR);
+            LOGGER.info(RECORDWITHID + deployStatusId + " selected from DeployStatus by User:" + principal.getName());
+        }catch (Exception e) {
+            LOGGER.error("error occurred while uploading file", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error occurred while uploading file" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)